examples/examples.py

```diff
@@ -554,3 +554,112 @@ def stft_plot(signal, sample_rate=SAMPLE_RATE):
 
 stft_plot(fm_out.detach().numpy())
 ipd.Audio(fm_out.detach().numpy(), rate=fm_vco.sample_rate.item())
+# -
+
+# ### Filters
+
+# +
+from ddspdrum.torchmodule import TorchMovingAverage, FIRLowPass
+
+# Create some noise to filter
+duration = 2
+noise = torch.tensor(np.random.rand(int(44100 * duration)) * 2 - 1).float()
+stft_plot(noise.detach().numpy())
+# -
+
+# **Moving Average Filter**
+#
+# A moving average filter is a simple finite impulse response (FIR) filter that calculates that value of a sample by taking the average of M input samples at a time. The filter_length defines how many samples M to include in the average.
+
```

Comment:
As I mentioned in the fb chat, the output here looks like and sounds like comb filter and not an MA filter. Not sure about the guts of it; and maybe I'm off... but would expect an LP type behaviour.

I might be totally off base. Lmk.